src/app/home/home.page.ts

```diff
@@ -0,0 +1,41 @@
+import { Component, OnInit, TemplateRef, ViewChild  } from '@angular/core';
+import { IonicSlides } from '@ionic/angular';
+import { register } from 'swiper/element/bundle';
+import { HttpClient } from '@angular/common/http';
+register();
+@Component({
+  selector: 'app-home',
+  templateUrl: './home.page.html',
+  styleUrls: ['./home.page.scss'],
+})
+export class HomePage implements OnInit {
+  swiperModules = [IonicSlides];
+  recent: any[] = [];
+  jsonData:any;
+  typeTemplateMapping: { [key: string]: TemplateRef<any> } = {};
+  @ViewChild('bannerTemplate') bannerTemplate!: TemplateRef<any>;
+  @ViewChild('productTemplate') productTemplate!: TemplateRef<any>;
+  @ViewChild('recommendationTemplate') recommendationTemplate!: TemplateRef<any>;
+
+  constructor(private http: HttpClient) { 
+  }
+
+  ngOnInit() {
+    this.http.get<any>('assets/data.json').subscribe(data => {
```

Comment:
why are we making http call for getting the form config data? not not from env file? or did we check with backend team for a form config API using which we will get this data?